seqr/utils/elasticsearch/es_search.py

```diff
@@ -1466,9 +1479,9 @@ def _annotations_filter(vep_consequences):
     return consequences_filter
 
 
-def _dataset_type_for_annotations(annotations, new_svs=False):
+def _dataset_type_for_annotations(annotations, new_svs=False, screen=False):
     sv = new_svs or bool(annotations.get('structural')) or bool(annotations.get('structural_consequence'))
-    non_sv = any(v for k, v in annotations.items() if k != 'structural' and k != 'structural_consequence')
+    non_sv = bool(screen) or any(v for k, v in annotations.items() if k != 'structural' and k != 'structural_consequence')
```

Comment:
so once we add that for SVs we will want to update the code here. But since there are no plans at least until 2023 to add this for SVs, I don't want us to do tons of futile searches against SVs in the meantime